### ISSUE TITLE
Failing mounts using EGI config repository and Kernel < 4.18

### ISSUE THREAD

**Issue by giffels** [cvmfs-contrib/cvmfsexec#3]
Hi @DrDaveD,

I have encountered a problem to mount repositories when using the EGI config repository and kernel version < 4.18.

Commands executed after installation:

```bash
makedist egi
makedist -o cvmfs_setup.sh
```

Doing a `cvmfs_exec.sh cms.cern.ch -- test.sh` returns the following output and the exit code is 141.

```
CernVM-FS: loading Fuse module... done
CernVM-FS: mounted cvmfs on /scratch/giffels/.cvmfsexec/dist/cvmfs/config-egi.egi.eu
Unmounting /scratch/giffels/.cvmfsexec/dist/cvmfs/config-egi.egi.eu
```

The test.sh looks like the following
```
#!/bin/bash +x
echo "Starting test.sh"
ls /cvmfs/cms.cern.ch
```
and is not executed at all.

I assume the problem is related to https://github.com/cvmfs-contrib/cvmfsexec/blob/da9bdf92947f2b939d631f9d5c12dfd15fe44cb5/cvmfsexec#L147-L158.

Putting debug output in those lines, I found that the EGI config repository is mounted first and afterwards the `cms.cern.ch` repository is mounted, but it fails with exit code 141. I guess mounting `cms.cern.ch` expects the egi repository to be present at /cvmfs and therefore fails, since the bind mount is created only later in the script. 

BTW. everything works as expected when using `makedist default` instead.

Could you have a look, please?

Thanks and best regards,
Manuel

**Comment by DrDaveD** [comment]
Thank you for the report, Manuel!

cvmfsexec does not expect the config repo to be at /cvmfs as you supposed, because the mountrepo command relocates the configuration paths to expect everything at the alternate path.

However, I was able to reproduce the problem and found that it was because of a relatively recent change in the egi config repo only, since I last tested cvmfsexec with egi (the problem doesn't occur with osg).  I have a fix in #4 and I cut a new release [cvmfsexec-2.4](https://github.com/cvmfs-contrib/cvmfsexec/releases/tag/v2.4).

**Comment by giffels** [comment]
Thank you Dave for the fix. I would like to confirm, that everything is working now with egi.
